### ISSUE TITLE
Package deployed to Maven but DB update failed

### ISSUE THREAD

**Issue by Deraen** [clojars/clojars-web#700]
Following package was deployed to Maven but the DB was not updated.

Could be there is also a problem with exception reporting:

> tcrawley [14:40]
@juhoteperi Ah, it looks like a database update failed (that's what the "Serialized task failed" response means). But I don't see an exception in sentry, so we are failing to send it there.

https://clojars.org/repo/cljsjs/html2canvas/1.0.0-alpha.12-0/
https://clojars.org/cljsjs/html2canvas
https://circleci.com/gh/cljsjs/packages/3450

```
Signing html2canvas-1.0.0-alpha.12-0.jar...
Deploying html2canvas-1.0.0-alpha.12-0.jar...
Sending html2canvas-1.0.0-alpha.12-0.jar.asc to https://clojars.org/repo/ (1k)
Sending html2canvas-1.0.0-alpha.12-0.pom.asc to https://clojars.org/repo/ (1k)
Sending html2canvas-1.0.0-alpha.12-0.jar to https://clojars.org/repo/ (104k)
Sending html2canvas-1.0.0-alpha.12-0.pom to https://clojars.org/repo/ (1k)
Retrieving maven-metadata.xml from https://clojars.org/repo/
Sending maven-metadata.xml to https://clojars.org/repo/ (1k)
Could not transfer metadata cljsjs:html2canvas/maven-metadata.xml from/to clojars (https://clojars.org/repo/): Access denied to: https://clojars.org/repo/cljsjs/html2canvas/maven-metadata.xml, ReasonPhrase: Forbidden - Serialized task failed.
                                                    java.lang.Thread.run                    Thread.java:  748
                      java.util.concurrent.ThreadPoolExecutor$Worker.run        ThreadPoolExecutor.java:  624
                       java.util.concurrent.ThreadPoolExecutor.runWorker        ThreadPoolExecutor.java: 1149
                                     java.util.concurrent.FutureTask.run                FutureTask.java:  266
                                                                     ...                                     
                                     clojure.core/binding-conveyor-fn/fn                       core.clj: 1938
                                                       boot.core/boot/fn                       core.clj: 1031
                                                     boot.core/run-tasks                       core.clj: 1021
                            cljsjs.boot-cljsjs.packaging/eval230/fn/G/fn                  packaging.clj:   97 (repeats 2 times)
                                          boot.task.built-in/fn/fn/fn/fn                   built_in.clj:  627 (repeats 2 times)
                           cljsjs.boot-cljsjs.packaging/eval297/fn/fn/fn                  packaging.clj:  194
                                          boot.task.built-in/fn/fn/fn/fn                   built_in.clj:  567
                                          boot.task.built-in/fn/fn/fn/fn                   built_in.clj:  864
                           cljsjs.boot-cljsjs.packaging/eval427/fn/fn/fn                  packaging.clj:  316
                                          boot.task.built-in/fn/fn/fn/fn                   built_in.clj: 1045
                                                       boot.pod/call-in*                        pod.clj:  413
                                                                     ...                                     
             org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke    ClojureRuntimeShimImpl.java:  102
             org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke    ClojureRuntimeShimImpl.java:  109
                                                                     ...                                     
                                                       boot.pod/call-in*                        pod.clj:  410
                                                   boot.pod/eval-fn-call                        pod.clj:  359
                                                      clojure.core/apply                       core.clj:  646
                                                                     ...                                     
                                                      boot.aether/deploy                     aether.clj:  354
                                                                     ...                                     
                                      cemerick.pomegranate.aether/deploy                     aether.clj:  407
                                                      clojure.core/apply                       core.clj:  646
                                                                     ...                                     
                            cemerick.pomegranate.aether/deploy-artifacts                     aether.clj:  334
                                                                     ...                                     
        org.sonatype.aether.impl.internal.DefaultRepositorySystem.deploy   DefaultRepositorySystem.java:  443
                org.sonatype.aether.impl.internal.DefaultDeployer.deploy           DefaultDeployer.java:  211
                org.sonatype.aether.impl.internal.DefaultDeployer.deploy           DefaultDeployer.java:  305
        org.sonatype.aether.connector.wagon.WagonRepositoryConnector.put  WagonRepositoryConnector.java:  475
org.sonatype.aether.connector.wagon.WagonRepositoryConnector$PutTask.run  WagonRepositoryConnector.java:  811
       org.apache.maven.wagon.providers.http.AbstractHttpClientWagon.put   AbstractHttpClientWagon.java:  497
       org.apache.maven.wagon.providers.http.AbstractHttpClientWagon.put   AbstractHttpClientWagon.java:  517
       org.apache.maven.wagon.providers.http.AbstractHttpClientWagon.put   AbstractHttpClientWagon.java:  523
       org.apache.maven.wagon.providers.http.AbstractHttpClientWagon.put   AbstractHttpClientWagon.java:  541
       org.apache.maven.wagon.providers.http.AbstractHttpClientWagon.put   AbstractHttpClientWagon.java:  617
org.apache.maven.wagon.authorization.AuthorizationException: Access denied to: https://clojars.org/repo/cljsjs/html2canvas/maven-metadata.xml, ReasonPhrase: Forbidden - Serialized task failed.
     org.sonatype.aether.transfer.MetadataTransferException: Could not transfer metadata cljsjs:html2canvas/maven-metadata.xml from/to clojars (https://clojars.org/repo/): Access denied to: https://clojars.org/repo/cljsjs/html2canvas/maven-metadata.xml, ReasonPhrase: Forbidden - Serialized task failed.
      metadata: #object[org.apache.maven.repository.internal.VersionsMetadata 0x45f3c3d0 "cljsjs:html2canvas/maven-metadata.xml"]
    repository: #object[org.sonatype.aether.repository.RemoteRepository 0x416d56f2 "clojars (https://clojars.org/repo/, releases+snapshots)"]
         org.sonatype.aether.deployment.DeploymentException: Failed to deploy metadata: Could not transfer metadata cljsjs:html2canvas/maven-metadata.xml from/to clojars (https://clojars.org/repo/): Access denied to: https://clojars.org/repo/cljsjs/html2canvas/maven-metadata.xml, ReasonPhrase: Forbidden - Serialized task failed.
                                 clojure.lang.ExceptionInfo: Failed to deploy metadata: Could not transfer metadata cljsjs:html2canvas/maven-metadata.xml from/to clojars (https://clojars.org/repo/): Access denied to: https://clojars.org/repo/cljsjs/html2canvas/maven-metadata.xml, ReasonPhrase: Forbidden - Serialized task failed.
    line: 51
```

**Comment by tobias** [comment]
This should get fixed when we merge and deploy #736, since it removes serialized writes.

**Comment by tobias** [comment]
We no longer have serialized writes (as of Clojars 79), so this should no longer be an issue.
